staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto

```diff
@@ -545,6 +551,113 @@ message ListPodSandboxResponse {
     repeated PodSandbox items = 1;
 }
 
+message PodSandboxStatsRequest {
+    // ID of the pod for which to retrieve stats.
+    string pod_sandbox_id = 1;
+}
+
+message PodSandboxStatsResponse {
+    PodSandboxStats stats = 1;
+}
+
+// PodSandboxStatsFilter is used to filter containers.
+// All those fields are combined with 'AND'
+message PodSandboxStatsFilter {
```

Comment:
Can we reuse `PodSandboxFilter` instead of creating new `PodSandboxStatsFilter`?

Reply:
I followed the example of ContainerStatsFilter, so I don't think we want to drop PodSandboxStatsFilter